htdocs/install/mysql/tables/llx_societe.sql

```diff
@@ -104,6 +104,8 @@ create table llx_societe
   fk_barcode_type          integer NULL   DEFAULT 0,                    -- barcode type
   price_level              integer NULL,                        		-- level of price for multiprices
   outstanding_limit	       double(24,8)   DEFAULT NULL,					-- allowed outstanding limit
+  id_risk              integer NULL,                        		-- risk level of outstanding
```

Comment:
What is a risk level of outstanding of a third party ? Do you have example ?

Reply:
There are loyal customers and some no, there are good payers and others are not ...
For example, I have a third of an average level of risk and who makes a purchase order of 60% of its limit, the commercial must ask his manager before validating his order or inquire with the bank about the situation current customer accounts to make the right decision ...
Other use cases is to classify a customer at a risk to raise is not to accept any more orders before that balance his credits!

For the limit of the days, there are good customers but it delivers checks with deadlines, one must have privileges for some by granting him to pay with a deadline of 3 months max and other customers whom they do not deserve to benefit than 15 days because they benefit from a discount ceiling so a reduced margin ...
These are examples, there are many others, they are indicators for the decision-making by the commercial and the manager!

Reply:
But what would be the different value of "id_risk" ? A percentage ?
If goal to to block order when outstanding is 60% of outstanding_limit, why not just setting the outstanding limit to 60% of outstanding_limit ?
The goal of outstanding limit is already to lock order if limit is raised.  Currently it show only a warning but a hidden option may lock it completely if user is not inside an advanced permission.